### PR TITLE
Update server.md to add http server  options for server

### PR DESCRIPTION
**Pull request by kdhamric**
Update server.md to add http server  options for server, including TRACETEST_SERVER_HTTPPORT and TRACETEST_SERVER_PATHPREFIX

This PR...

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Review by schoren — APPROVED** [review]
Looks good! I would also add the config file version of those settings:

```yaml
server:
  httpPort: 11633
  pathPrefix: /
```

**Review by jorgeepc — APPROVED** [review]

File: docs/docs/configuration/server.md [hunk 1]
```diff
@@ -26,6 +26,11 @@ The list of environment variables and example values is:
 - `TRACETEST_POSTGRES_PASSWORD: "postgres"`
 - `TRACETEST_POSTGRES_PARAMS: ""`
 
+You can also change the defaults for the Tracetest server, altering the port and path you access the dashboard from. For a docker based install done locally, this url is typically http://localhost:11633/. You can alter it by setting either of these environmental variables:
```
Comment (adnanrahic) [inline docs/docs/configuration/server.md:29]:
```suggestion
You can also change the defaults for the Tracetest server, altering the port and path you access the dashboard from. For a Docker-based install done locally, this URL is typically `http://localhost:11633/`. You can alter it by setting either of the environment variables or using the `server` object in the server configuration file:
```

**Review by adnanrahic — APPROVED** [review]
Added a suggestion to include a config file example as well.

**Review by jfermi — APPROVED** [review]

File: docs/docs/configuration/server.md [hunk 2]
```diff
@@ -26,6 +26,11 @@ The list of environment variables and example values is:
 - `TRACETEST_POSTGRES_PASSWORD: "postgres"`
 - `TRACETEST_POSTGRES_PARAMS: ""`
 
+You can also change the defaults for the Tracetest server, altering the port and path you access the dashboard from. For a docker based install done locally, this url is typically http://localhost:11633/. You can alter it by setting either of these environmental variables: 
+
```
[the remark this answers is not in the document]
Comment (kdhamric) [reply docs/docs/configuration/server.md:30]: I am going to add this as a second pr...

**Pull request merged**